public/main.css

```diff
@@ -1,3 +1,5 @@
+@import url('https://fonts.googleapis.com/css?family=Muli&display=swap');
```

Comment:
Can you rather import via HTML, more performant.

Reply:
Yeah will do, was just playing around with a different font :smile: